chain/app/genesis.go

```diff
@@ -153,6 +153,8 @@ func GetDefaultDataSourcesAndOracleScripts(owner sdk.AccAddress) json.RawMessage
 			oracleScript.name,
 			oracleScript.description,
 			code,
+			"schema",
```

Comment:
You can add schema and code URL in oracleScripts (line 124)

Reply:
Example schema of "Crypto price script (Borsh version" schema should be `{"Input": "{ \\"kind\\": \\"struct\\", \\"fields\\": [ [\\"symbol\\", \\"string\\"], [\\"multiplier\\", \\"u64\\"] ] }", "Output": "{ \\"kind\\": \\"struct\\", \\"fields\\": [ [\\"px\\", \\"u64\\"] ] }`